src/js/components/Chart/doc.js

```diff
@@ -201,14 +209,14 @@ export const themeDoc = {
         responsiveBreakpoint: 'small',
     }`,
   },
-  'global.graph.colors': {
-    description: 'The color to use when not specified via color.',
-    type: '[string] or { dark: [string], light: [string] }',
-  },
   'global.opacity': {
     description: 'The opacity of the Chart stroke.',
     type: 'string',
```

Comment:
```suggestion
    type: 'string | number',
```

Reply:
Actually, `global.opacity` is an object.